src/index.html

```diff
@@ -6,7 +7,71 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+  <!--endregion-->
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-fri calendar--month-length-29">
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
```

Comment:
It is not necessary to add an empty line here between blocks
```suggestion
      <div class="calendar-day"></div>
      <div class="calendar-day"></div>
```